src/common/ui-components/Menu/Menu.module.scss

```diff
@@ -1,58 +1,28 @@
-@import '../../../styles/partials/variables';
-@import '../../../styles/partials/mixins';
-
-$menu-z-index: 1000;
+@import '../../../styles/spacings';
+@import '../../../styles/mixins';
+@import '../../../styles/z-index';
+@import '../../../styles/icons';
 
 .container {
```

Comment:
I usually prefer having a dark className for the container and 
```SCSS
.container {
   &.dark {
      .icon {
```
and inside I define all the CSS values I want to override instead of having two versions of everything like `logo` and `logoDark`, that would also make assigning classNames in the component simpler.

Reply:
Done.